config/routes.rb

```diff
@@ -20,6 +20,7 @@
     get '/app/accounts/:account_id/settings/inboxes/new/microsoft', to: 'dashboard#index', as: 'app_new_microsoft_inbox'
     get '/app/accounts/:account_id/settings/inboxes/new/:inbox_id/agents', to: 'dashboard#index', as: 'app_twitter_inbox_agents'
     get '/app/accounts/:account_id/settings/inboxes/new/:inbox_id/agents', to: 'dashboard#index', as: 'app_microsoft_inbox_agents'
+    get '/app/accounts/:account_id/settings/inboxes/:inbox_id', to: 'dashboard#index', as: 'app_microsoft_inbox_settings'
```

Comment:
Why do we need this?

Reply:
The original callback redirected to the `app_microsoft_inbox_agents` this is the onboarding page where the admin is asked to add agents. The new route is for the inbox settings page, the user is redirected here when reauthorized, I can rename it to `app_inbox_settings` since it is not Microsoft specific